Tests/Tests.asmdef

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
First time I come across this file type - what is it and how is it used.

Reply:
Puts the tests in their own assembly definition/namespace. I believe the purpose is to keep test code inaccessible from the rest, much like the UnityEditor namespace